### DISCUSSION TITLE
What is your process for using this?

### DISCUSSION THREAD

**Discussion by Johnson90512** [firefly-iii/firefly-iii#6649] [General]
Hey Guys.

Just like the title says, what do you see as the process for using this software? Is it more of a all manual or all automated or somewhere in between? If it is somewhere in between, how do you manage to use recurring transactions as well as importing your bank information?

Thank you

**Comment by JC5** [comment]
So nobody replied yet so let me share my own process.

I started out doing everything manually, but lately I've switched to the data importer with Nordigen. I have to clean up most of my transactions but that's OK. I use some recurring transactions for accounts I can't import automatically tho.

**Reply by Johnson90512** [reply]
In your experience, does the data importer seem to be more for people outside of the US? It doesn't appear to me that the  US banks are supported. I've been doing manually, but I think the data importer would be sweet.

Also, thank you for your reply and the program!

**Reply by JC5** [reply]
They used to be supported by Salt Edge, but lack of legislation means that there is no unified API or mandatory 3rd party access for financial institutions. That means, in turn, that most organizations have given up trying to connect to all of them.
